python/ray/tune/result_grid.py

```diff
@@ -80,6 +81,36 @@ def get_best_result(
             )
         )
 
+    def get_dataframe(
```

Comment:
Add test?

Reply:
Functionality is tested in `test_experiment_analysis.py` and we only pass through here. We can do that but maybe should separate this out from this PR.

Reply:
Sounds good!